ui/app/AppLayouts/Wallet/popups/SignPhraseModal.qml

```diff
@@ -1,113 +0,0 @@
-import QtQuick 2.15
-import QtQuick.Controls 2.15
-import QtQuick.Layouts 1.15
-
-import utils 1.0
-
-import StatusQ.Core.Theme 0.1
-import StatusQ.Controls 0.1
-
-import shared.panels 1.0
-import shared.popups 1.0
-import "../stores"
-
-// TODO: replace with StatusModal
-ModalPopup {
-    id: signPhrasePopup
-
-    signal remindLaterClicked()
-    signal acceptClicked()
-
-    title: qsTr("Signing phrase")
-    height: 390
-    closePolicy: Popup.NoAutoClose
-
-    Column {
-        anchors.left: parent.left
-        anchors.right: parent.right
-
-        StyledText {
-            anchors.horizontalCenter: parent.horizontalCenter
-            text: qsTr("This is your signing phrase")
-            font.pixelSize: 17
-            font.weight: Font.Bold
-            horizontalAlignment: Text.AlignHCenter
-            height: Theme.padding * 3
-        }
-
-        StyledText {
-            anchors.horizontalCenter: parent.horizontalCenter
-            text: qsTr("You should see these 3 words before signing each transaction")
-            font.pixelSize: 15
-            width: 330
-            wrapMode: Text.WordWrap
-            horizontalAlignment: Text.AlignHCenter
-            height: Theme.padding * 4
-        }
-
-        Rectangle {
-            color: Theme.palette.baseColor2
-            height: 44
-            width: parent.width
-            StyledText {
-                id: signingPhrase
```

Comment:
We should remove al of this from the store and backend also in case this is final?